spec/unit/classes/server/config_spec.rb

```diff
@@ -163,82 +276,99 @@ class { 'postgresql::server': }
     end
   end
 
-  describe 'on Gentoo' do
+  describe 'with managed pg_hba_conf and ipv4acls' do
     let(:pre_condition) do
       <<-EOS
         class { 'postgresql::globals':
           version => '9.5',
         }->
-        class { 'postgresql::server': }
+        class { 'postgresql::server':
+          manage_pg_hba_conf => true,
+          ipv4acls => [
+            'hostnossl all all 0.0.0.0/0 reject',
+            'hostssl all all 0.0.0.0/0 md5'
+          ]
+        }
       EOS
     end
     let(:facts) do
       {
         os: {
-          family: 'Gentoo',
-          name: 'Gentoo',
-          release: { 'full' => 'unused' },
+          family: 'RedHat',
+          name: 'CentOS',
+          release: {
+            'full'  => '7.9.2009',
+            'major' => '7',
+            'minor' => '9',
+          },
           selinux: { 'enabled' => true },
         },
-        osfamily: 'Gentoo',
+        concat_basedir: tmpfilename('server'),
```

Comment:
[concat_basedir is dead](https://github.com/puppetlabs/puppetlabs-postgresql/commit/3fba56bd8ab36e4862f9ea7b2e9cc69206271531)
```suggestion
```